symphony/bdk/core/config/model/bdk_config.py

```diff
@@ -0,0 +1,55 @@
+from symphony.bdk.core.config.model.bdk_bot_config import BdkBotConfig
+from symphony.bdk.core.config.model.bdk_server_config import BdkServerConfig
+from symphony.bdk.core.config.model.bdk_client_config import BdkClientConfig
+from symphony.bdk.core.config.model.bdk_ssl_config import BdkSslConfig
+
+
+class BdkConfig(BdkServerConfig):
+    """Class containing the Bdk configuration """
+
+    def __init__(self, **kwargs):
```

Comment:
Please put directly constructor params `scheme`, `host`, `port`, etc. before **kwargs and remove `kwargs.get`
The code below works:
```python
def _private(a, b, c, **kwargs):
    print(f'a={a}, b={b}, c={c}')
    print(kwargs)

def test_unpack():
    d = {'a': "value a", 'b': "value b", 'c': "value c", 'd': 'toto'}
    _private(**d)
```

Reply:
After thinking about it, (And please tell me if I overthought this one) but what if the dictionary looks like this: (removed c value)
```python
d = {'a': "value a", 'b': "value b", 'd': 'toto'}
```
This will raise the error: `TypeError: _private() missing 1 required positional argument: 'c'`

<br>

I don't think this is a use case we want. This means if we write our code like this:
```python
class BdkConfig(...)
def __init__(self, host, context, port, agent, pod, ...,  **kwargs):
```

and don't provide the `context` at the root of the config file such as the minimalist one:
```yaml
host: acme.symphony.com                                    
bot: 
    username: bot-username                                  
    privateKey:
      path: /path/to/bot/rsa-private-key.pem
```
We will get this error, `TypeError: __init__() missing 1 required positional argument: 'context'` which is not something we want, right ?

Reply:
What about `context = None` in the parameters ?